shell_sort/Cpp/shell_sort.cpp

```diff
@@ -0,0 +1,62 @@
+#include <iostream>
+
+using std::cout;
+
+void insertion_sort(int a[], int a_size) {
+    int i = 0;
+    while (i < a_size) {
+        int j = i;
+        while (j > 0 && a[j-1] > a[j]) {
+            // swap
+            int temp = a[j-1]; 
```

Comment:
Line ends in whitespace.  Consider deleting these extra spaces.  [whitespace/end_of_line] [4]

*Origin: CPPLintBear, Section: `all.cpplint`.*